### ISSUE TITLE
Do not cross-require tables between BUFR and GRIB

### ISSUE THREAD

**Issue by manfredsc** [wmo-im/GRIB3#3]
See https://github.com/wmo-im/BUFR4/issues/84

BUFR4 Code Table 0-02-020 (satellite classification) is also used to decode GRIB2 PDT's 4.30 to 4.35.
This should be detangled so common code tables are in CCT.
Like the related tables C-5 and C-8, BUFR4 Code Table 0-02-020 should be moved to become an own CCT entry.

**Comment by sebvi** [comment]
I fully agree that all tables used by both formats should be made part of the common code tables.
